CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Support for computed members
+
+### Changed
+- Test suite to be complete
+
```

Comment:
we are missing the breking changes of dropping support for previous major terraform and provider versions..

please also mark commits and PR as breaking using conventional commits `!` marker

Reply:
Thanks @mariux,  Suggestions have been addressed.